WordPress/Classes/Models/AbstractPost.m

```diff
@@ -281,6 +282,27 @@ - (Media *)featuredImage
 
 #pragma mark - Convenience methods
 
+- (BOOL)isDraft
+{
+    if ([self.status isEqualToString:PostStatusDraft]) {
```

Comment:
Really minor, but we could join both if conditions with an "or" operator (since both cases return `YES`).